.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "library"]
 	path = library
 	url = https://github.com/alexander-mead/library
+	branch = f90wrap
```

Comment:
I guess that this can be removed now that your `f90wrap` has been merged with the library `master` branch?

Reply:
Done